assets/dev/scss/editor/panel/controls/_globals.scss

```diff
@@ -219,6 +219,13 @@
 			}
 		}
 	}
+
+	&__repeater-separator {
```

Comment:
1. You can do it as a border-top of '.elementor-control-type-global-style-repeater .elementor-button-wrapper' instead of creating a new element.

2. What about dark mode?

Reply:
1. In this case it won't be 260px width

2. Good point - I didn't get the variable name for dark mode. I will estimate for now